inc/spbc-settings.php

```diff
@@ -4017,6 +4080,69 @@ function spbc_get_key_auto($direct_call = false)
     die(json_encode($out));
 }
 
+function spbc_settings__update_account_email($direct_call = false)
+{
+    if ( ! $direct_call) {
+        spbc_check_ajax_referer('spbc_secret_nonce', 'security');
+    }
+
+    global $spbc;
+
+    $account_email = Post::get('accountEmail');
+
+    // not valid email
+    if (!$account_email || !filter_var(Post::get('accountEmail'), FILTER_VALIDATE_EMAIL)) {
+        die(
+            json_encode(
+                array(
+                    'error' => 'Please, enter valid email.'
+                )
+            )
+        );
+    }
+
+    // protection against accidental request from a child site in the shared account mode
+    if (!is_main_site() && isset($spbc->network_settings['multisite__work_mode']) && $spbc->network_settings['multisite__work_mode'] != 3) {
+        die(
+            json_encode(
+                array(
+                    'error' => 'Please, enter valid email.'
```

Comment:
The same error? Probably accidentally copied text.

Reply:
Fixed, thanks